shapeshift/web
# feat: actually lazy load hdwallet adapter modules

## Description

Yo dawg, we heard you like wallets, so we put some lazy wallet loading in your wallet loading, so you can transact while you transact! 🚀 Welcome to our super-duper PR where we're turning sloth-mode ON to hit you up with the snazziest, laziest import of hdwallet npm packages that you've ever seen. 

Why, you ask? Because ain't nobody got time to wait for those hefty hdwallet bundles to load! 😴 By incorporating a luxuriously lazy loading mechanism, we make sure your user experience is smoother than a fresh jar of peanut butter. 🥜✨ Now you can keep your transactions and crypto dealings as swift as a ninja, all while our code does the heavy (or should we say, light?) lifting in the background.

In a nutshell:
- Maximize efficiency 🚀
- Minimize loading times ⏲️
- Optimize your chill while we work in the stealth 🐢👤

## Pull Request Type


## Issue (if applicable)



- tackles https://github.com/shapeshift/web/issues/5445 
 
## Risk



Very high. Test all wallets connection and reconnection, in all states, e.g native wallet import, locked native unlock, MM in locked state, KK wallet import, KK unlock, app refresh, etc...
Test all states and ensure no regression against prod.

Tested with:
- KK ✅ 
- Native ✅ 
- MM ✅ 
- WCV2 ✅
- Keplr 🚫 borked, but also borked in prod (see https://github.com/shapeshift/hdwallet/pull/646 for fix)

## Testing



- See risks - test all wallet states and ensure no regression against prod
- Ensure wallet/accounts switching is happy and has no regressions against prod

### Engineering



- ☝🏽 

### Operations



- ☝🏽 

## Screenshots (if applicable)
